sound/soc/intel/boards/sof_sdw.c

```diff
@@ -253,12 +253,21 @@ static struct sof_sdw_codec_info codec_info_list[] = {
 	},
 };
 
-static inline int find_codec_info_part(unsigned int part_id)
```

Comment:
Shouldn't we modify all the tables to add the version number above? Otherwise you haven't really tested the code?

Reply:
> Shouldn't we modify all the tables to add the version number above? Otherwise you haven't really tested the code?

Currently, all the existing codec infos are for all version id. I will add the version id for those codec info which is for a specific version only in the follow up patch.